src/modules/regions/entities/region.entity.ts

```diff
@@ -0,0 +1,37 @@
+import { Entity, PrimaryGeneratedColumn, Column, CreateDateColumn, UpdateDateColumn } from 'typeorm';
```

Comment:
Please keep your code DRY by extending the base entity class.
Then remove the created_at, updated_at and I'd fields from this definition afterwards